packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -193,13 +193,15 @@ export const ChannelView: FC = () => {
     }
   }, [clearAllFilters, currentTabName, setIsFiltersOpen])
   const mappedChannelNftCollectors =
-    channelNftCollectors?.map(({ amount, member }) => ({
-      nftsAmount: amount,
-      url: member?.metadata.avatar?.__typename === 'AvatarUri' ? member?.metadata.avatar?.avatarUri : '',
-      tooltipText: member?.handle,
-      onClick: () => navigate(absoluteRoutes.viewer.member(member?.handle)),
-      memberUrl: absoluteRoutes.viewer.member(member?.handle),
-    })) || []
+    channelNftCollectors
+      ?.filter(({ channel }) => channel?.id !== id)
+      .map(({ amount, member }) => ({
+        nftsAmount: amount,
+        url: member?.metadata.avatar?.__typename === 'AvatarUri' ? member?.metadata.avatar?.avatarUri : '',
+        tooltipText: member?.handle,
+        onClick: () => navigate(absoluteRoutes.viewer.member(member?.handle)),
+        memberUrl: absoluteRoutes.viewer.member(member?.handle),
+      })) || []
```

Comment:
What exactly are you trying to do here? If I understand the code correctly, this `.filter(({ channel }) => channel?.id !== id)` will have no effect. In line 105, we provide a filter to fetch only collectors for this specific channel so all the entries in that list will have that channel ID. What is the goal here?

Reply:
Yeah, you are right, this is bullsh**t. I was trying to resolve the same problem we had in NftWidget and NftTile.
My channel has 1 NFT and CollectorsBox shows wrong owner.

![Zrzut ekranu 2022-10-5 o 09 45 13](https://user-images.githubusercontent.com/13706246/194007959-5da0bdce-6b73-45a4-8aab-b0a5189aa218.png)